modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -204,6 +209,40 @@ def get_facility_timezone_memoized(facility_location_id)
         facility_info[:timezone]&.[](:time_zone_id)
       end
 
+      def normalize_eps_appointment(appt)
+        {
+          id: appt[:id].to_s,
+          status: appt[:state] == 'submitted' ? 'booked' : 'proposed',
+          patientIcn: appt[:patientId],
+          created: appt.dig(:appointmentDetails, :lastRetrieved),
+          requestedPeriods: [
+            {
+              start: appt.dig(:appointmentDetails, :start),
+              end: calculate_end_time(appt.dig(:appointmentDetails, :start))
+            }
+          ].compact,
+          locationId: appt[:locationId],
+          clinic: appt[:clinic],
+          contact: appt[:contact]
+        }.compact
+      end
+
+      def calculate_end_time(start_time)
+        return nil unless start_time
+
+        Time.zone.parse(start_time) + 60.minutes
+      end
+
+      def merge_appointments(new_appointments, basic_appointments)
+        normalized_new = new_appointments[:appointments].map { |appt| normalize_eps_appointment(appt) }
+        basic_data = basic_appointments[:data].is_a?(Array) ? basic_appointments[:data] : [basic_appointments[:data]]
```

Comment:
I verified with Simi that this will not break any other logic (and nothing in the controller) when pushing back a single item in an array (not for frontend either)